gtest_parallel.py

```diff
@@ -1,5 +1,5 @@
 #!/usr/bin/env python2
-# Copyright 2013 Google Inc. All rights reserved.
+# Copyright 2017 Google Inc. All rights reserved.
```

Comment:
I don't think this should be changed. Semantically this is the same file, so do 2013.